### ISSUE TITLE
Suppor dns ?

### ISSUE THREAD

**Issue by Dazmed707** [qH0sT/AndroSpy#44]
support dns for conecction ? (no-ip)
example :::> example.ddns.net

**Comment by qH0sT** [comment]
yes, ofcourse.. please dont open unnecessary issue. Instead of this, ask me question: asuslaptop230gmail.com. when I avaible, I can answer you.

new update is coming sooon..
